deployment/terraform/create.go

```diff
@@ -69,6 +72,8 @@ func New(cfg *deployment.Config) *Terraform {
 // Create creates a new load test environment.
 func (t *Terraform) Create() error {
 	err := t.preFlightCheck()
+	defer os.RemoveAll(t.dir)
```

Comment:
I understand the need for this contraption. It's because the directory is created inside `init` which is called from `preFlightCheck`. So then we would have to bubble up the dir name two levels up.

Instead what I would suggest is to have an exposed method `(*terraform).Destroy`, which executes

```
if t.dir != "" {
os.RemoveAll(t.dir)
}
```

And then this gets called in a defer block from `RunCreateCmdF` and `RunDestroyCmdF` itself. That way, it becomes less error-prone and easy to use. As somebody writing a new terraform action won't need to remember to add a os.RemoveAll call.